packages/venia-concept/.graphqlconfig

```diff
@@ -11,11 +11,15 @@
                         "apollo",
                         "literal"
                     ],
+                    "magentoBackendEdition": "${env:MAGENTO_BACKEND_EDITION}",
                     "filesGlob": "../{peregrine,venia-ui,venia-concept}/{lib,src}/**/*.{js,graphql,gql}",
+                    "ceFilesGlob": "../{peregrine,venia-ui,venia-concept}/{lib,src}/**/*.ce.js",
+                    "eeFilesGlob": "../{peregrine,venia-ui,venia-concept}/{lib,src}/**/*.ee.js",
                     "ignore": [
                         "../venia-ui/lib/**/giftOptions.js",
                         "../venia-ui/lib/**/wishlistPage.gql.js",
-                        "../venia-ui/lib/**/wishlistItem.gql.js"
+                        "../venia-ui/lib/**/wishlistItem.gql.js",
+                        "../venia-ui/lib/**/giftCardQueries.js"
```

Comment:
Not sure why the GraphQL validator is failing on this file. The queries and mutations used in this file are valid ones according to the schema docs but for some reason, the validator doesn't like them. So ignoring them for now.

Reply:
GiftCards are an EE only feature. It looks like what we did here is selectively render `CartPage/PriceAdjustments/giftCardSection`, so the that queries file didn't need to have the flavor extension. I think it would be preferable here to maybe rename `giftCardQueries.js => giftCardQueries.ee.js` to keep validation working for EE.

Reply:
Didn't consider that rename is going to make editors think this file doesn't exist now. Do we have a pattern to solve this? Do we require that the non flavored file must also exist?

![Screen Shot 2021-03-16 at 2 06 57 PM](https://user-images.githubusercontent.com/462953/111367369-ae0df780-8662-11eb-9375-fac5ed28884f.png)

Reply:
I think it is editor-specific. I don't see it happening on VS Code. Right now we just create files with flavors attached (CE or EE) but we don't have non-flavored ones to accommodate for editors. If we want to solve this, it has to be its own ticket.

![image](https://user-images.githubusercontent.com/35203638/111488346-26c68f80-8707-11eb-941a-c86f6252361b.png)